Logging/RockLib.Logging.Analyzers.Test/AssemblySettings.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Xunit;
```

Comment:
I noticed this file has been added in the repos you've been working on.  Is this something we should be including with all the libraries as we go through updating them?